test/metabase/search/filter_test.clj

```diff
@@ -65,8 +65,8 @@
                       [:in :model_id [1 2 3 4]]
                       [:in :model ["card" "dataset" "metric" "dashboard" "action"]]
                       [:= :search_index.archived true]
-                      [:>= [:cast :search_index.created_at :date] #t"2024-10-01"]
-                      [:< [:cast :search_index.created_at :date] #t"2024-10-02"]
+                      [:>= [:cast :search_index.model_created_at :date] #t"2024-10-01"]
+                      [:< [:cast :search_index.model_created_at :date] #t"2024-10-02"]
```

Comment:
woah I did not get your comment at the beginning of this pr and now I suddenly got it, haha